cluster/ansible/playbooks/roles/master/templates/kube-apiserver.jsonnet

```diff
@@ -0,0 +1,94 @@
+function(cfg)
+  {
+    apiVersion: "v1",
+    kind: "Pod",
+    metadata: {
+      name: "kube-apiserver",
+      namespace: "kube-system",
+      labels: {
+        tier: "control-plane",
+        component: "kube-apiserver",
+      },
+    },
+    spec: {
+      hostNetwork: true,
+      containers: [
+        {
+          name: "kube-apiserver",
+          image: "%(docker_registry)s/kube-apiserver:%(kubernetes_version)s" % cfg.cluster,
+          resources: {
+            requests: {
+              cpu: "250m",
+            },
+          },
+          command: [
+            "/bin/sh",
+            "-c",
+            |||
+              /usr/local/bin/kube-apiserver \
+                --address=127.0.0.1 \
+                --etcd-servers=http://127.0.0.1:4001 \
+                --cloud-provider=gce \
+                --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,ResourceQuota \
```

Comment:
I don't believe it will block privileged containers if you `--allow-privileged` on api-server + kubelet. However, I do believe this causes some conformance tests failures based on "run-as". But haven't had the chance to dig in.